gui/dialogs/aircraft-dialog.xml

```diff
@@ -297,6 +302,38 @@
 
                     <checkbox>
                         <halign>left</halign>
+                        <label>Place control lock</label>
+                        <property>sim/model/c172p/cockpit/control-lock-placed</property>
+                        <live>true</live>
+                        <enable>
+                            <property>/sim/model/c172p/securing/allow-securing-aircraft</property>
+                            <greater-than>
+                                <property>sim/model/c172p/cockpit/yoke-aileron</property>
+                                <value>-0.05</value>
+                            </greater-than>
+                            <less-than>
+                                <property>sim/model/c172p/cockpit/yoke-aileron</property>
+                                <value>0.05</value>
+                            </less-than>
+                            <greater-than>
+                                <property>sim/model/c172p/cockpit/yoke-elevator</property>
+                                <value>-0.05</value>
+                            </greater-than>
+                            <less-than>
+                                <property>sim/model/c172p/cockpit/yoke-elevator</property>
+                                <value>0.05</value>
+                            </less-than>
```

Comment:
again, these conditionals above concerning aileron and elevator position maybe should be removed